scripts/generate_download_page.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env /bin/bash
+
+###############################################################################
+# Copyright (c) 2024 Ericsson
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+
+# Bash version of the old "rcp_index_php". Contrary to the original php script, 
+# this one is run only once and generates a static HTML page. The script will 
+# output on stdout - redirect the output to a file in the wanted folder.
+#
+# CLI parameters:
+# 1): Folder where the TC packages are already deployed. Can be relative or absolute.
+#     Used to list the release packages and construct the download page with their
+#     name and sizes
+# 2): HTTP path: path relative to the Eclipse download server web root directory. 
+#     Starts with "/tracecompass/" and used to construct the download link
+# 3): Release (version), e.g. "10.1.0" 
+#
+# Usage example:
+# $ ./generate_download_page.sh /path/to/release/folder /tracecompass/releases/10.1.0/rcp/ 10.1.0 > /path/to/release/folder/index.html
+
+# it might be possible to simplify the script's input parameters once we've played with it
+
+# examples of web server path vs linux path: 
+# https://www.eclipse.org/downloads/download.php?file=/tracecompass/stable/rcp/trace-compass-10.1.0-20240918-1731-linux.gtk.x86_64.tar.gz
+# https://www.eclipse.org/downloads/download.php?file=/tracecompass/master/rcp/trace-compass-10.2.0-20241128-0440-linux.gtk.x86_64.tar.gz
+# https://www.eclipse.org/downloads/download.php?file=/tracecompass/releases/10.1.0/rcp/trace-compass-10.1.0-20240916-1358-linux.gtk.x86_64.tar.gz
+# /home/data/httpd/download.eclipse.org/tracecompass/stable/rcp
+# /home/data/httpd/download.eclipse.org/tracecompass/
+
+
+dir=$1
+http_path=$2
+title="Trace Compass Release $3"
```

Comment:
The download page can be include Nightly builds. The title should be different in that case.
Currently we have the following 3 different strings:
- Trace Compass Latest Stable Version
- Trace Compass RCP Nightly
- Trace Compass Release 10.1.0

Reply:
We can streamline the title to make it easy to generate from the Jenkins jobs.

Reply:
So, probably we want to pass the title when calling the script? i.e. the 3rd CLI param becomes "title"?

Reply:
I've made this change in the latest version